access/email/config.go

```diff
@@ -183,10 +189,29 @@ func (c *Config) CheckAndSetDefaults() error {
 		c.Log.Severity = "info"
 	}
 
-	// Validate emails in user aliases
-	for _, e := range c.Delivery.Recipients {
-		if !lib.IsEmail(e) {
-			return trace.BadParameter("Invalid email address %v in users.recipients", e)
+	if len(c.Delivery.Recipients) > 0 {
+		if len(c.RoleToRecipients) > 0 {
+			return trace.BadParameter("provide either delivery.recipients or role_to_recipients, not both.")
+		}
+
+		c.RoleToRecipients = config.RecipientsMap{
+			types.Wildcard: c.Delivery.Recipients,
+		}
+		c.Delivery.Recipients = nil
+	}
+
+	if len(c.RoleToRecipients) == 0 {
+		return trace.BadParameter("missing required value role_to_recipients.")
+	}
+	if len(c.RoleToRecipients[types.Wildcard]) == 0 {
+		return trace.BadParameter("missing required value role_to_recipients[%v].", types.Wildcard)
```

Comment:
@marcoandredinis @Joerger I am curious about this check - is ignoring some role requests not an expected scenario? For example, imagine I use 2 plugins, slack and email, and I want some requests to only go through email (or vice versa).

Currently it seems like you always have to have a "catch all" matcher. Which may be fine, I'm just wondering if this scenario was considered before.

Reply:
I'm not sure if this scenario was considered, but it was a requirement in the design conversation - https://github.com/gravitational/teleport-plugins/issues/411#issuecomment-1033107396. It might make more sense to remove it and give users that option.

@russjones was there a downside you had in mind to allowing users to omit the wildcard entry?